dolphin/api/v1/storages.py

```diff
@@ -11,17 +11,72 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import copy
 
+from oslo_log import log
 from six.moves import http_client
 import webob
+from webob import exc
 
+from dolphin import db, context
 from dolphin.api.common import wsgi
 
+LOG = log.getLogger(__name__)
+
+def build_storages(storages):
```

Comment:
Hi @NajmudheenCT  could you please format the code?

Reply:
done, plz check